plugin/pkg/admission/serviceplan/defaultserviceplan/admission_test.go

```diff
@@ -64,6 +65,19 @@ func newFakeServiceCatalogClientForTest(sc *servicecatalog.ServiceClass) *fake.C
 	fakeClient.AddReactor("list", "serviceclasses", func(action core.Action) (bool, runtime.Object, error) {
 		return true, scList, nil
 	})
+
+	// react to the given plans
+	spList := &servicecatalog.ServicePlanList{
+		ListMeta: metav1.ListMeta{
```

Comment:
do we need to set the resource version here?

Reply:
I copied it from up above.